### PR TITLE
Start page UI

### PR DESCRIPTION
**Pull request by fishyvishy**
The start page is completed, with all registration and login functionality implemented with adherence to clean architecture. Message Dialogs have been added to improve user experience. When the remaining UI pages are completed, the login will have to be modified to allow the user to enter the Search Page UI.

### CODE REVIEW

File: src/main/java/UI/startPageUI/registerUserView.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public registerUserView(startPageViewModel viewModel, CreateUserandFridgeControl
 
         this.register_response = new JTextField(30);
         this.register_response.setEditable(false);
-        this.register_response.setOpaque(true);
+        this.register_response.setOpaque(false);
```
Comment (EricKarpovits) [inline src/main/java/UI/startPageUI/registerUserView.java:40]: is setOpaque always going to be false?

**Review by EricKarpovits — APPROVED** [review]
Great work @fishyvishy

**Review by rickychen175 — APPROVED** [review]
Looks good. Register and login works well.

**Review by RedFoxity — APPROVED** [review]
Looks good!

**Pull request merged**